ctpolicy/ctpolicy.go

```diff
@@ -276,9 +130,13 @@ func (ctp *CTPolicy) getOperatorSCTs(ctx context.Context, cert core.CertDER, exp
 		res := <-results
 		if res.err != nil {
 			errs = append(errs, res.err.Error())
+			if res.url != "" {
+				ctp.winnerCounter.WithLabelValues(res.url, failed).Inc()
+			}
```

Comment:
Arguably we could even remove this bit and only increment for the two logs which are successful -- if the log returned an error, does it really count as a winner? But I'm totally fine either way.

Reply:
I honestly don't know enough to know whether it would be helpful to know if a given log is failing?? It believe it would have to fail pretty fast to beat the winners. @mcpherrinm do you have an opinion here?

Reply:
I think knowing that a given log is failing is useful, but we can get that from the publisher. The issue here is that the data is unreliable in a way -- it would tell us that a given log is failing, but only at the sampling rate of "when we randomly select that log to be in the first two, and then it fails quickly and not slowly so we get its failed result before getting two proper SCTs".